samples/matter/template/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -22,4 +22,4 @@ CONFIG_CHIP_FACTORY_DATA=n
 CONFIG_CHIP_FACTORY_DATA_WRITE_PROTECT=n
 
 # Set SUIT template paths
-CONFIG_SUIT_ENVELOPE_TEMPLATE="suit_templates/app_envelope.yaml.jinja2"
+CONFIG_SUIT_ENVELOPE_TEMPLATE="../common/suit_templates/app_envelope.yaml.jinja2"
```

Comment:
I have a doubt about it. We rather try not to use relative paths based on the current location (../), but e.g. we use some macros in cmake (like ZEPHYR_NRF_MODULE_DIR). The problem with the current approach is that it will not work with the copying sample feature of the vs code, because someone may want to copy sample to other location than matter/samples/...